ui/public/scripts/components/mobile-app/view-app-config.component.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
+ * @ngdoc component
+ * @name mcp.component:view-app-config
+ * @description
+ * # view-app-config
+ */
+angular.module('mobileControlPanelApp').component('viewAppConfig', {
+  template: `<div>
+              <div class="row app-config">
```

Comment:
Could we add some inline help text (beside/below) each of these form inputs?

Reply:
What sort of help text to you envision here? Each value has a descriptive label and we get here from the create screen which has inline help and validation

Reply:
@sedroche Nevermind. I realise my mistake as this is the viewAppConfig. I thought this was the Create/Edit app config.